public/stylesheets/style.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
CSS and not SCSS?

Reply:
CSS file was auto generated by express-generator.
Changing to SCSS is part of https://github.com/software-developer-org/website/issues/8